deployment/cloud/main.tf

```
@@ -0,0 +1,47 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.61"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-west-2"
+  assume_role {
+    role_arn = "arn:aws:iam::871683513797:role/supportsphere-deploy"
+    session_name = "supportsphere-infra-deployment"
+    external_id = "supportsphere-infra-deployment"
+  }
+
+  // Tags all resources created from this provider with {"Project": "Support Sphere", "Neighborhood": "Laurelhurst"}
+  default_tags {
+    tags = {
+      Project = "Support Sphere",
+      Neighborhood = "Laurelhurst"
+    }
+  }
+}
+
+module "server" {
+  source = "./modules/server"
+  
+}
+
+resource "aws_resourcegroups_group" "support_sphere_group_laurelhurst" {
```

```suggestion
resource "aws_resourcegroups_group" "this" {
```

Following the terraform naming best practice: https://www.terraform-best-practices.com/naming#resource-and-data-source-arguments

Since there's only one resource of this type, the label should be `this`

---

Oh, I didn't know about this! Definitely can do, and thank you for the resource!